sdcm/sla/sla_tests.py

```diff
@@ -123,12 +123,25 @@ def validate_role_service_level_attributes_against_db():
                 role_for_attach.attach_service_level(new_service_level)
                 role_for_attach.validate_role_service_level_attributes_against_db()
 
-                # Workaround for issue https://github.com/scylladb/scylla-enterprise/issues/2572
-                if restart_scylla:
-                    nodes_for_restart = [node for node in tester.db_cluster.nodes if node.jmx_up() and node.db_up()
-                                         and not node.running_nemesis]
-                    tester.db_cluster.restart_binary_protocol(nodes=nodes_for_restart)
-                # End workaround
+                for node in tester.db_cluster.nodes:
+                    if node.db_up():
+                        node.remoter.run(
+                            "curl -X POST --header 'Content-Type: application/json' --header 'Accept: application/json'"
+                            " http://127.0.0.1:10000/service_levels/switch_tenants"
+                        )
+
+                time.sleep(5*60)
```

Comment:
what we are waiting for here? Could you please leave a comment?  Can we make some smart wait? for example by periodically checking service_levels/count_connections or it is not the case here?